modules/apps/27-interchain-accounts/controller/client/cli/tx.go

```diff
@@ -0,0 +1,62 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/controller/types"
+)
+
+const (
+	// The controller chain channel version
+	flagVersion = "version"
+)
+
+// NewTxCmd creates and returns the tx command
+func NewTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                        "controller",
+		Short:                      "ica controller transactions subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	cmd.AddCommand(
+		newRegisterAccountCmd(),
+	)
+
+	return cmd
+}
+
+func newRegisterAccountCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "register [connection-id]",
+		Short: "Register account via connection end identifier on the controller chain",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			connectionID := args[0]
+			owner := clientCtx.GetFromAddress().String()
+			version, err := cmd.Flags().GetString(flagVersion)
+			if err != nil {
+				return err
+			}
+
+			msg := types.NewMsgRegisterAccount(connectionID, owner, version)
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	cmd.Flags().String(flagVersion, "", "Controller chain channel version")
+	flags.AddTxFlagsToCmd(cmd)
```

Comment:
It might be nice if we added documentation on how to create versions with ics29 enabled. Maybe in the long section

Reply:
I would be fine with doing this in a later PR when we can test everything out e2e

Reply:
Can create an issue for this and add it later too.

Reply:
Great ideas y'all. Lets open an issue